p4-16/psa/examples/psa-example-hello-world.p4

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2019 Cisco Systems, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <core.p4>
+#include "../psa.p4"
```

Comment:
Why do we need `..` in the path?

Reply:
All P4_16 files in the p4-16/psa/examples have this, to force them to include the psa.p4 file in this repository.  At least when we started, there was no psa.p4 file in p4c at all.

If we want to make future changes to psa.p4, having this ../psa.p4 for these example files ensures that 'make check' will use the locally modified psa.p4, not the sure-to-be-older one in p4c.